docker/run_setup.sh

```diff
@@ -81,10 +81,14 @@ if [ ! -e $dir_prefix/NETWORK_RUNNING ]; then
 
     export PATH="$PATH:$HOME/.foundry/bin"
 
-    subnet_a_url="http://127.0.0.1:9650/ext/bc/$subnet_a_chain_id/rpc"
-    subnet_b_url="http://127.0.0.1:9650/ext/bc/$subnet_b_chain_id/rpc"
-    subnet_c_url="http://127.0.0.1:9650/ext/bc/$subnet_c_chain_id/rpc"
-    c_chain_url="http://127.0.0.1:9650/ext/bc/C/rpc"
+    subnet_a_rpc_url="http://127.0.0.1:9650/ext/bc/$subnet_a_chain_id/rpc"
+    subnet_a_ws_url="ws://127.0.0.1:9650/ext/bc/$subnet_a_chain_id/ws"
```

Comment:
are these webscoket urls used anywhere, or is it added just incase we use in future?

Reply:
They're not used anywhere in this script (like most of the variables actually), but they are written out to `var.sh` so that they can be easily used by other scripts/containers if needed.